build_in_functions_task_all_func.py

```diff
@@ -0,0 +1,12 @@
+# Implement your own all function
+
+def custom_func(input_data):
+    for element in input_data:
+        if not element:
+            return False
+    return True
```

Comment:
It is ok. 
Another solution checks if False in input data. So you do not need to loop